README.md

```diff
@@ -129,6 +129,7 @@ _Environment variable denoted in parentheses._
 * `-P, --project [path]` Path to TypeScript JSON project file (`TS_NODE_PROJECT`)
 * `-D, --ignoreDiagnostics [code]` Ignore TypeScript warnings by diagnostic code (`TS_NODE_IGNORE_DIAGNOSTICS`)
 * `-O, --compilerOptions [opts]` JSON object to merge with compiler options (`TS_NODE_COMPILER_OPTIONS`)
+* `-F, --transformers [path]` Path of .js transformers to pass to TypeScript (`TS_NODE_TRANSFORMERS`)
```

Comment:
I'm not comfortable making `-F` the flag for `transformers` here - would prefer to leave it short-flag-less.